test/dataframes.jl

```diff
@@ -69,6 +69,40 @@ df2 = @byrow df begin
     end
 end
 
+df = DataFrame(A = 1:3, B = [2, 1, 2])
+
```

Comment:
```suggestion
```

Reply:
needs to be in global scope because of `@eval`.